recipes-multimedia/imx-vpuwrap/imx-vpuwrap_4.5.5.bb

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2013-2016 Freescale Semiconductor
```

Comment:
@thochstein could you find out the logs?

Reply:
```
r60874@tx30imx-01:~/git/vpu_wrapper$ git log --oneline --no-decorate 8a3f1b0..b683295
b683295 MA-17072 [VC8000E] Set AVC default profile/level
ae24b72 MA-16984 update Android.bp to remove "imx_defaults"
1349ea2 MA-16873-4 Update vpu_wrapper path
c1235d2 MA-16830-2 Fix memleak when free memory in wrapper
f680b3c MMFMWK-8745 hantro: enable reorder config for hantro vpu
3685f06 MA-16717-3 add VPU_DEC_SKIP for CODEC_PIC_SKIPPED
35b43ea MA-16717-2 Add Config to reset codecdata flag
8c92057 MMFMWK-8742 Map decoders to timestamps
ad0bc7f MMFMWK-8739 [8mp]Grecorder: Only one recorded ts file play well when set '-n 3'
6c306bc MMFMWK-8738 [8mp] Set right input buffer's Y,U,V physical address
1524207 MMFMWK-8736 Fix hantro decoder can't get fullrange flag of files encoded with vc8000e
19fe757 MA-16593 [evk_8mm] Fix video hang due to error frame
13459f7 MMFMWK-8733 Update EULA.txt for vpu_wrapper according to imx-release-info
35ea5e6 [configurea.ac] Update version to 4.5.5 for next release
94c38de MA-16435-1 Modify Android makefile
96e189f MMFMWK-8720 Replace docs/EULA.txt and add SRC file SCR-imx-vpuwrap.txt
341dc9b MMFMWK-8720 Simplify vpu wrapper
bf6680e MMFMWK-8720 Sync vpu wrapper code with Android and change copyright
1e6bc61 MMFMWK-8720 fix vc8000e encoder can't work
a61d115 MMFMWK-8722 Fix color abnormal issue of streaming feature
f68972d MMFMWK-8722 [8mp] Fix bitrate getting smaller issue of VC8000E encoder
e7b3992 MMFMWK-8718 Enable cropping for hantro encoder at preprocess stage
017ecf7 MMFMWK-8714 [8mp] Adjust parameters to get correct filesize with given bit rate.
95acceb MMFMWK-8595 seperate 845 and 865 rootfs
180d19f MMFMWK-8595 Enable VC8000E encoder for 865
6fb749b [configurea.ac] Update version to 4.5.4 for next release
```